spec/support/migration_name_helper.rb

```diff
@@ -0,0 +1,6 @@
+def require_migration
```

Comment:
at the time of requiring the migration, or referencing the migration class name, the migration_spec_helper is not in context.